products/storage/api.yaml

```
@@ -780,4 +780,60 @@ objects:
         name: timeUpdated
         api_name: updated
         description: The modification time of the object metadata.
-
+  - !ruby/object:Api::Resource
+    name: 'HmacKey'
+    kind: 'storage#hmacKey'
+    base_url: projects/{{project}}/hmacKeys
+    create_url: projects/{{project}}/hmacKeys?serviceAccountEmail={{serviceAccountEmail}}
+    self_link: projects/{{project}}/hmacKeys/{{accessId}}
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': 'hhttps://cloud.google.com/storage/docs/authentication/managing-hmackeys'
+      api: 'https://cloud.google.com/storage/docs/json_api/v1/projects/hmacKeys'
+    description: |
+      The hmacKeys resource represents an HMAC key within Cloud Storage. The resource
+      consists of a secret and HMAC key metadata. HMAC keys can be used as credentials
+      for service accounts.
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'serviceAccountEmail'
+        description: The email address of the key's associated service account.
+        input: true
+        required: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'secret'
+        output: true
+        description: 'HMAC secret key material.'
```

```suggestion
        description: |
          HMAC secret key material.
```

Mind using this format for descriptions?